react/components/SearchBar/components/SearchBar.js

```diff
@@ -97,6 +112,24 @@ const SearchBar = ({
     [navigate, attemptPageTypeSearch, customSearchPageUrl]
   )
 
+  const validateInput = () => {
+    if (minSearchTermLength && inputValue.length < minSearchTermLength) {
+      return intl.formatMessage({
+        id: 'store/search.searchTermTooShort',
```

Comment:
Use `defineMessages` instead of passing the id directly. When we use `defineMessages` we got some static analyzes from `builder-hub`.